internal/cmd/cmd.go

```diff
@@ -370,6 +373,10 @@ func NewRootCmd() *cobra.Command {
 	return rootCmd
 }
 
+func addNonInteractiveFlag(flags *pflag.FlagSet, bind *bool) {
+	flags.BoolVar(bind, "non-interactive", false, "Disable all prompts for input")
+}
```

Comment:
I moved this out of `rootOptions` to be an option that we can add to any command that has interactive prompts. That way we don't have to pass around options from the root command.

Reply:
In a subsequent commit I also set a default here based on "is stdin a terminal", which removes the need to call other functions later to determine the interactivity mode, and allows us to override the value when necessary.